ui/app/components/job-status/panel/steady.js

```diff
@@ -185,6 +185,10 @@ export default class JobStatusPanelSteadyComponent extends Component {
     return this.job.allocations.filter((a) => !a.isOld && a.hasBeenRestarted);
   }
 
+  get runningAllocs() {
+    return this.job.allocations.filter((a) => a.clientStatus === 'running');
+  }
```

Comment:
`@job.runningAllocs` comes from the job's summary as defined [here](https://github.com/hashicorp/nomad/blob/86ee56b8c58b543a245a83bf8776eacebe297c9d/ui/app/models/job.js#L406) in case you were curious